access/src/main/external/static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -288,6 +301,17 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities', 'co
 		});
 	};
 	
+	//For edit label modal
+	ResultObjectActionMenu.prototype.editLabel = function(resultObject) {
+		var editLabelForm = new EditLabelForm({
+			alertHandler : this.options.alertHandler,
+			actionHandler : this.actionHandler,
+			resultObject : resultObject
+		});
+		editLabelForm.open(resultObject.metadata.id, resultObject.metadata);
```

Comment:
See note above about changing open() to take a resultObject. For consistency with other AbstractFileUploadForm subclasses we should consider creating only one instance of EditLabelForm and calling open() on that.